src/components/Rules/styled.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+
+export const Rule = styled.div`
+  z-index: 10;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Once you change the text to use p tags, we can set the font-family of elements in p tags. Maybe we can do this globally in `index.css` too.